### PR TITLE
Update the 1Password SCIM Bridge to v1.3.0

### PR DESCRIPTION
**Pull request by grellyd**

### CODE REVIEW

**Comment by paulanunda** [comment]
@grellyd I'll review this today, thanks for the update!

File: stacks/op-scim-bridge/yaml/op-scim-bridge.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
     spec:
       containers:
       - name: op-scim-bridge
-        image: 1password/scim:v1.2.1
+        image: 1password/scim:v1.3.0
```
Comment (paulanunda) [inline stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: I'm seeing image pull issues for this image; does it exist?

Comment (grellyd) [reply stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: Thanks Paul. You are correct. This version is oddly named. It should be `1password/scim:v1.3.0-2`. I think I was accidentally testing with a cached older version actually titled `1password/scim:v1.3.0` which never saw the light of day.

Comment (grellyd) [reply stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: I believe I have fixed that correctly. After pruning my old images and re-rendering the DO marketplace stack, it deploys correctly to my DO cluster. 

Let me know how that looks.

**Pull request merged**